shell/platform/darwin/ios/ios_external_view_embedder.h

```diff
@@ -13,14 +13,14 @@ namespace flutter {
 class IOSExternalViewEmbedder : public ExternalViewEmbedder {
  public:
   IOSExternalViewEmbedder(
-      const std::shared_ptr<PlatformViewsController>& platform_views_controller,
+      __weak FlutterPlatformViewsController* platform_views_controller,
       const std::shared_ptr<IOSContext>& context);
 
   // |ExternalViewEmbedder|
   virtual ~IOSExternalViewEmbedder() override;
 
  private:
-  const std::shared_ptr<PlatformViewsController>& platform_views_controller_;
+  __weak FlutterPlatformViewsController* platform_views_controller_;
```

Comment:
same here and above

Reply:
This is correct; ownership was always weak -- notice this is a `const std::shared_ptr<PlatformViewsController>&`, not a `std::shared_ptr<PlatformViewsController>`.

Mentioned in the patch description but this was totally an unobvious trap waiting to catch all of us, and really unclear code. Now it's much clearer that this is weak ownership.